bigquery/snippets/table/integration_test.go

```diff
@@ -150,6 +150,12 @@ func TestTables(t *testing.T) {
 	if err := updateTableAddColumn(tc.ProjectID, testDatasetID, testTableID); err != nil {
 		t.Fatalf("updateTableAddColumn(%q %q): %v", testDatasetID, testTableID, err)
 	}
+
+	// Change tables to avoid hitting metadata update limits in a short period.
+	testTableID, err = bqtestutil.UniqueBQName("testtable")
+	if err := createTableExplicitSchema(tc.ProjectID, testDatasetID, testTableID); err != nil {
```

Comment:
Do we need to clean up the extra table?

Reply:
No.  There's a recursive delete of the encapsulating dataset as part of a defer, which cleans up all the resources.